### PR TITLE
NetworkManager-openvpn :  update to 1.8.18 + gtk4 support

### PR DESCRIPTION
**Pull request by hsnfirdaus**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-LIBC)

### CODE REVIEW

**Review by kruceter — CHANGES_REQUESTED** [review]
Replace `sed` in the template with `vsed`.

Change your commit message header to:

`NetworkManager-openvpn: update to 1.8.18, add gtk4 support.`

Rebase the master branch against your own instead of merging.

**Comment by kruceter** [comment]
@hsnfirdaus, squash these two commits together. Do not forget to alter the commit message the way I asked you to.

**Comment by hsnfirdaus** [comment]
@kruceter should i resolve this conflict too?

**Comment by kruceter** [comment]
> should i resolve this conflict too?

Rebase your branch with recent changes from master.

```
git checkout master
git pull --rebase upstream master
git rebase master nm-openvpn
```

Push again after that.

**Comment by hsnfirdaus** [comment]
> ```
> git rebase master nm-openvpn
> ```

@kruceter how about this message when i run it: 
```
...
Rebasing (131/160618)
Rebasing (132/160618)
Auto-merging .gitignore
CONFLICT (add/add): Merge conflict in .gitignore
error: could not apply 16ae7bdf63... Added a .gitignore file to ignore default created dirs.
hint: Resolve all conflicts manually, mark them as resolved with
hint: "git add/rm <conflicted_files>", then run "git rebase --continue".
hint: You can instead skip this commit: run "git rebase --skip".
hint: To abort and get back to the state before "git rebase", run "git rebase --abort".
Could not apply 16ae7bdf63... Added a .gitignore file to ignore default created dirs.
```

**Comment by kruceter** [comment]
Try to skip them all, @hsnfirdaus.

**Comment by paper42** [comment]
That's an issue in your checkout because you modified .gitignore for some reason, but it's not in a commit. Abort this, make sure you don't need your local changes (git diff), remove them (git restore .) and try again.

**Comment by hsnfirdaus** [comment]
> That's an issue in your checkout because you modified .gitignore for some reason, but it's not in a commit. Abort this, make sure you don't need your local changes (git diff), remove them (git restore .) and try again.

Well i ended up this conflict by re clone my repo and rebase...

Thank you @paper42 @kruceter

File: srcpkgs/NetworkManager-openvpn/template [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 post_extract() {
-	sed -i 's/\(#define NM_OPENVPN_\(USER\|GROUP\)\).*/\1 "_nm_openvpn"/' \
+	vsed -i 's/\(#define NM_OPENVPN_\(USER\|GROUP\)\).*/\1 "_nm_openvpn"/' \
 		shared/nm-service-defines.h
 }
 
 pre_configure() {
 	autoreconf -fi
-}
+}
```
Comment (paper42) [inline srcpkgs/NetworkManager-openvpn/template:27]: don't delete the end of the line here

File: srcpkgs/NetworkManager-openvpn/template [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+configure_args="--disable-static --with-gtk4"
 hostmakedepends="automake gettext-devel glib-devel intltool libtool pkg-config"
-makedepends="gtk+3-devel libglib-devel libnma-devel libsecret-devel"
+makedepends="gtk+3-devel libglib-devel libnma-devel libsecret-devel gtk4-devel"
 depends="openvpn"
 short_desc="NetworkManager VPN plugin for OpenVPN"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://wiki.gnome.org/Projects/NetworkManager"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
```
Comment (paper42) [inline srcpkgs/NetworkManager-openvpn/template:14]:
```suggestion
distfiles="${GNOME_SITE}/NetworkManager-openvpn/${version%.*}/NetworkManager-openvpn-${version}.tar.xz"
```

**Pull request merged**